src/cli.c

```diff
@@ -176,7 +176,7 @@ static void send_header(quicly_stream_t *stream, int is_http1, int status, const
 
 static int flatten_file_vec(quicly_sendbuf_vec_t *vec, void *dst, size_t off, size_t len)
 {
-    int fd = (int)vec->cbdata;
+    int fd = (uint64_t)vec->cbdata;
```

Comment:
Would you mind clarifying the warning that you are seeing here?

I do not see why casting into `uint64_t` then implicitly casting to `int` is better than directly casting to `int`.

Reply:
I'm getting the following warning : 
`src/cli.c:179:14: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]` 
as vec->cbdata is a `void*` it's complaining to have to cast & change from 64bits to 32bits at the same time (on a 64bits system). But you're right, I might have gone a bit quick on that one, we would be better off with a `intptr_t`

btw I'm using `cmake version 3.10.2` and `Make 4.1` on a 64bit ubuntu and running `cmake . && make`